packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_CoarseOperator_decl.hpp

```diff
@@ -167,16 +167,16 @@ namespace FROSch {
         mutable XMultiVectorPtr YCoarseSolve_;
         mutable XMultiVectorPtr YCoarseSolveTmp_;
 
-        ConstXMapPtrVecPtr GatheringMaps_;
+        ConstXMapPtrVecPtr GatheringMaps_ = ConstXMapPtrVecPtr(0);
         XMapPtr CoarseSolveMap_;
 
         SubdomainSolverPtr CoarseSolver_;
 
         ParameterListPtr DistributionList_;
 
-        XExportPtrVecPtr CoarseSolveExporters_;
+        XExportPtrVecPtr CoarseSolveExporters_ = XExportPtrVecPtr(0);
 #ifdef FROSCH_COARSEOPERATOR_EXPORT_AND_IMPORT
```

Comment:
Was this issue (with reverse-mode Export not doing what you wanted) ever fixed?  If so, then we can get rid of this macro.